epictrack-web/src/components/workPlan/event/EventForm.tsx

```diff
@@ -679,6 +679,10 @@ const EventForm = ({
               <ETFormLabel required>{anticipatedLabel}</ETFormLabel>
               <ControlledDatePicker
                 name="anticipated_date"
+                disabled={Boolean(
+                  selectedConfiguration?.id &&

```

Comment:
One more condition is required if you want to disable the datepicker. The workphase should be legislated